pkg/apis/configuration/validation/validation.go

```diff
@@ -210,6 +210,24 @@ func validateUpstreamHealthCheck(hc *v1alpha1.HealthCheck, fieldPath *field.Path
 	return allErrs
 }
 
+func validateSessionCookie(sc *v1alpha1.SessionCookie, fieldPath *field.Path) field.ErrorList {
```

Comment:
Proper validation is missing:

* For name, use `isCookieName` function.
* For domain, use `IsDNS1123Subdomain`, although the leading `.` should be allowed.